app/models/search_builder.rb

```diff
@@ -32,27 +32,30 @@ def only_published_items(_permission_types = discovery_permissions, _ability = c
   end
 
   def limit_to_non_hidden_items(_permission_types = discovery_permissions, _ability = current_ability)
-    [policy_clauses,"(*:* NOT hidden_bsi:true)"].compact.join(" OR ")
+    [policy_clauses, "(*:* NOT hidden_bsi:true)"].compact.join(" OR ")
   end
 
   # Overridden to skip for admin users
   def add_access_controls_to_solr_params(solr_parameters)
-    if current_ability.cannot? :discover_everything, MediaObject
-      solr_parameters[:fq] ||= []
-      solr_parameters[:fq] << gated_discovery_filters.reject(&:blank?).join(' OR ')
-      avalon_solr_access_filters_logic.each do |filter|
-        solr_parameters[:fq] << send(filter, discovery_permissions, current_ability)
-      end
-      Rails.logger.debug("Solr parameters: #{solr_parameters.inspect}")
+    return unless current_ability.cannot? :discover_everything, MediaObject
+
+    solr_parameters[:fq] ||= []
+    solr_parameters[:fq] << gated_discovery_filters.reject(&:blank?).join(' OR ')
+    avalon_solr_access_filters_logic.each do |filter|
+      solr_parameters[:fq] << send(filter, discovery_permissions, current_ability)
     end
+    Rails.logger.debug("Solr parameters: #{solr_parameters.inspect}")
   end
 
   def search_section_transcripts(solr_parameters)
     return unless solr_parameters[:q].present? && SupplementalFile.with_tag('transcript').any? && !(blacklight_params[:controller] == 'bookmarks')
 
-    return if solr_parameters[:q].match?(/[\{\}]/)
-    # In order for the multi-word query to work we need to NOT escape the query AND replace the spaces as +; this is also true for quoted phrase searches
-    transcript_subquery = "transcript_tsim:#{solr_parameters[:q].gsub(/ /, '+')}"
+    # In order for the multi-word query to work we need to NOT RSolr.solr_escape the query AND replace the spaces as +; this is also true for quoted phrase searches
+    # We can manually escape solr special characters that cause issues.
+    query = solr_parameters[:q].gsub(/([(){}\[\]\^\*?:])/, "\\\\\1").tr(' ', '+')
+    # Quotes need special handling to allow phrase searches.
+    query = query.gsub(/"/, "\\\\\1") if query.count('"').odd?
```

Comment:
This appears to escape all quotes if there is an odd number of them.  So a stray unmatched quote at the end invalidates all quotes from making phrase searches?  Would it be better to escape just the last one or it that assuming too much about the query?

Reply:
I did some manual testing and I'm running into issues with certain queries all of which have matched quotes so shouldn't be getting escaped in this line.
```
test "item"
"test item" test
"test item
" 1997 TV AD Campaign", 30 spots, January 
```

Reply:
Hmmm...yeah, that is a good point about invalidating the phrase searches.

For the case of odd number of quotation marks, what do you think about not trying to escape them at all? We can catch the error and display a message about "All quotes in search must be matched".

As for your examples of stuff still failing, those look like they are giving a different error. The character issues return a "Cannot parse" error, whereas those examples are returning a "no field name specified in query and no default specified via 'df' param" error. I'm guessing that something about the combo of phrase search and multiword search is confusing solr. Unsure on how best to handle that...